tests/db/cli/test_flow.py

```diff
@@ -18,24 +21,60 @@ def test_flows_list(job_controller, four_jobs):
 
 
 def test_delete(job_controller, four_jobs):
+    from jobflow_remote.jobs.runner import Runner
+    from jobflow_remote.jobs.state import JobState
     from jobflow_remote.testing.cli import run_check_cli
 
+    # run one of the jobs to check that the output is not deleted
+    runner = Runner()
+    job_1_uuid = four_jobs[0].jobs[0].uuid
+    runner.run_one_job(job_id=(job_1_uuid, 1))
+    assert job_controller.get_job_doc(job_id=job_1_uuid).state == JobState.COMPLETED
+    assert job_controller.jobstore.get_output(job_1_uuid) == 6
+
     run_check_cli(
         ["flow", "delete", "-fid", four_jobs[0].uuid],
         required_out="Deleted Flow",
         cli_input="y",
     )
     assert job_controller.count_flows() == 1
+    assert job_controller.jobstore.get_output(job_1_uuid) == 6
 
-    # don't confirm
-    run_check_cli(["flow", "delete", "-fid", four_jobs[1].uuid], cli_input="n")
-    assert job_controller.count_flows() == 1
-
+    # run the command without returning any match
     run_check_cli(
         ["flow", "delete", "-fid", four_jobs[0].uuid],
         required_out="No flows matching criteria",
     )
 
+    # don't confirm and verbose option
+    # only check the first characters of the uuid because it may be cut in the output
+    outputs = ["This operation will delete the following 1 Flow", four_jobs[1].uuid[:5]]
+    run_check_cli(
+        ["flow", "delete", "-fid", four_jobs[1].uuid, "-v"],
+        required_out=outputs,
+        cli_input="n",
+    )
+    assert job_controller.count_flows() == 1
```

Comment:
I was puzzled why it was giving one, as I thought four_jobs was just one flow so I went to look into the conftest to undestand that it's actually 2 flows. Maybe it would be good to just rename the fixture so that it is more clear ?

Reply:
done